workshops/book-dash/book-dash-ldn-report.md

```diff
@@ -0,0 +1,122 @@
+# The Turing Way | Book Dash - London
+
+
+* Intro to Book Dash MCR slides: https://zenodo.org/record/3233655
```

Comment:
Could we follow best practice and use DOI links to link to zenodo rather than the URL please?